blockindex/liquidstaking_indexer.go

```diff
@@ -0,0 +1,950 @@
+// Copyright (c) 2023 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package blockindex
+
+import (
+	"context"
+	"math/big"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/iotexproject/go-pkgs/hash"
```

Comment:
new line

Reply:
i'm sorry to make multiple mistakes in the same issue. Use a tool to format it automatically should be better for developer. 
The `--local` flag of `goimports` tool could fix it, i have configured in my ide. Maybe we could also add it in pre-commit?